lib/LessParser.js

```diff
@@ -26,6 +27,7 @@ module.exports = class LessParser extends Parser {
     super.atrule(token);
     importNode(this.lastNode);
     variableNode(this.lastNode);
+    toString(this.lastNode);
```

Comment:
I don't understand why this was done

Reply:
I was trying to extend `AtRule` nodes with our own `toString`, but I'm hoping I can find a better solution.